components/cli/vendor/github.com/docker/docker/api/types/swarm/task.go

```diff
@@ -1,6 +1,10 @@
 package swarm
 
-import "time"
+import (
```

Comment:
Also brought in by PR:
* https://github.com/moby/moby/pull/33575 Plugins on swarm